### PR TITLE
(#116) Move Assertion2 -> Assertion

### PR DESCRIPTION
**Pull request by llorllale**
PR for #116:

Overwrote `Assertion` with `Assertion2`.

This finishes work on refactoring the `Assertion` API.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#117](https://github.com/llorllale/cactoos-matchers/issues/117) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#117](https://github.com/llorllale/cactoos-matchers/issues/117) is assigned to @ilyakharlamov[/z](https://www.0crat.com/u/ilyakharlamov), here is [why](https://www.0crat.com/footprint/C63314D6Z/f49cd6d4-22b6-46f0-9d4b-639b46c32390); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/llorllale/cactoos/matchers/AssertionTest.java [hunk 1]
```diff
@@ -44,7 +41,6 @@
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
```
Comment (ilyakharlamov) [inline src/test/java/org/llorllale/cactoos/matchers/AssertionTest.java]: It looks like JavadocMethodCheck can be removed as well here

File: src/test/java/org/llorllale/cactoos/matchers/InputHasContentTest.java [hunk 2]
```diff
@@ -46,7 +46,7 @@
     @Test
```
Comment (ilyakharlamov) [inline src/test/java/org/llorllale/cactoos/matchers/InputHasContentTest.java]: `StringLiteralsConcatenationCheck` can be removed here

File: src/test/java/org/llorllale/cactoos/matchers/InputHasContentTest.java [hunk 3]
```diff
@@ -46,7 +46,7 @@
...
     public void matchesInputContent() {
```
Comment (ilyakharlamov) [inline src/test/java/org/llorllale/cactoos/matchers/InputHasContentTest.java]: `Test case for {@link org.llorllale.cactoos.matchers.InputHasContent}.` can be replaced with `Test case for {@link InputHasContent}.` The javadoc compiler will take the full path from the package declaration anyway.

Comment (llorllale) [reply src/test/java/org/llorllale/cactoos/matchers/InputHasContentTest.java]: @ilyakharlamov fixed

File: src/test/java/org/llorllale/cactoos/matchers/AssertionTest.java (hunk 1 above)
Comment (llorllale) [reply src/test/java/org/llorllale/cactoos/matchers/AssertionTest.java]: @ilyakharlamov fixed

File: src/test/java/org/llorllale/cactoos/matchers/InputHasContentTest.java (hunk 2 above)
Comment (llorllale) [reply src/test/java/org/llorllale/cactoos/matchers/InputHasContentTest.java]: @ilyakharlamov fixed

**Comment by llorllale** [comment]
@ilyakharlamov please check

**Comment by ilyakharlamov** [comment]
@llorllale looks good to me

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 10 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @ilyakharlamov[/z](https://www.0crat.com/u/ilyakharlamov), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#117](https://github.com/llorllale/cactoos-matchers/issues/117) is now out of scope

**Comment by llorllale** [comment]
@rultor release, tag is `0.15`

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +25 point(s) just awarded to @ilyakharlamov[/z](https://www.0crat.com/u/ilyakharlamov)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Pull request merged**